### PR TITLE
Fixing readme issues

### PR DESCRIPTION
**Pull request by mbruzek**
Addresses the issues with the README #11, #12, #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,30 +30,44 @@ Note: This bundle is also conjure-up enabled. Refer to the
...
 update the status messages with progress, so it is recommended to run
-`watch juju status` to monitor the cluster coming up. After it is deployed you
-need to grab the kubectl and configuration from the Kubernetes leader node to
-control the cluster:
+`watch juju status` to monitor the cluster coming up.
 
-    juju scp kubernetes/0:kubectl_package.tar .
-    tar zxf kubectl_package.tar
-    ./kubectl get pods
+After the cluster is deployed you need to grab the kubectl binary and
```
Comment (chuckbutler) [inline README.md]: s/grab/download

File: README.md [hunk 2]
```diff
@@ -30,30 +30,44 @@ Note: This bundle is also conjure-up enabled. Refer to the
...
+configuration for your cluster from the Kubernetes **master unit** to control
+the cluster. To find the master unit check the `juju status` output or run
+a command on all kubernetes units to detect the leader:  
+
+    juju run --service kubernetes is-leader
```
Comment (chuckbutler) [inline README.md]: Didn't we put in nifty (leader) status messaging?

**Comment by chuckbutler** [comment]
a few nits left inline. Otherwise +1 LGTM

File: README.md (hunk 2 above)
Comment (mbruzek) [inline README.md]: @chuckbutler Yes we did. The master unit will prefix all status messages with "(master)".  @battlemidget had some questions about kubernetes leader/master today so I decided to put more words about finding the leader/master. Do you disagree with anything I wrote here?

**Comment by arosales** [comment]
@mbruzek thanks for the updates to the bundle README. These updates look to resolve #11, #12, #13.

+1 from me.

**Pull request merged**